sdk/eventhub/azure-eventhubs/azure/eventhub/aio/consumer_async.py

```diff
@@ -159,7 +185,7 @@ def queue_size(self):
             return self._handler._received_messages.qsize()
         return 0
 
-    async def receive(self, **kwargs):
+    async def receive(self, max_batch_size=None, timeout=None):
```

Comment:
`async def receive(self, *, max_batch_size=None, timeout=None)`

Reply:
it is fixed.